Dockerfile

```diff
@@ -11,14 +11,12 @@
 
 WORKDIR /app
 
-ENV PYTHONUNBUFFERED=1
+ENV PYTHONUNBUFFERED="default"
```

Comment:
The `PYTHONUNBUFFERED` environment variable should not be set to 'default'. It is typically set to '1' to ensure that Python output is sent straight to the terminal without being buffered.